verilog/analysis/symbol_table.h

```diff
@@ -54,6 +54,7 @@ enum class SymbolMetaType {
   kFunction,
   kTask,
   kStruct,
+  kEnum,
```

Comment:
It might be useful to distinguish between `kEnumType` and `kEnumValue` or `kEnumConstant`.

In `typedef enum { FOO } bar_t;` `bar_t` is the enum type, and `FOO` is a `kEnumValue`.